mapping/enable/async_loader.py

```diff
@@ -12,33 +13,43 @@ class AsyncLoader(HasTraits):
     def start(self):
         self._thread.start()
 
+    def stop(self):
+        self._stop_signal.set()
+        self._thread.join()
+
     def put(self, request):
         self._queue.put(request)
 
     ### Private interface ##################################################
     
     _thread = Any
+    _stop_signal = Any
```

Comment:
Should this be `Instance(Event)` (and I guess similarly with `_thread` being `Instance(Thread)`) or is there a reason we want to be more general than this?

In fact, `Instance(Event, ())` would mean that the default handler would not be needed.